vowpalwabbit/plt.cc

```diff
@@ -0,0 +1,382 @@
+// Copyright (c) by respective owners including Yahoo!, Microsoft, and
+// individual contributors. All rights reserved. Released under a BSD (revised)
+// license as described in the file LICENSE.
+#include <cfloat>
+#include <cmath>
+#include <cstdio>
+#include <sstream>
+#include <vector>
+#include <unordered_map>
+#include <unordered_set>
+#include <algorithm>
+#include <queue>
+#include "reductions.h"
+#include "vw.h"
+
+using namespace VW::LEARNER;
+using namespace VW::config;
+
+namespace plt_ns
```

Comment:
Does this `_ns` stand for namespace? If so, then I think keeping it as just `plt` is more consistent with elsewhere,

Reply:
Yes, I think `_ns` stand for namespace, I followed convention used in two other tree-based methods: `memory_tree.cc` and `recall_tree.cc`, they also use `_ns` suffix for their namespaces.